src/encoded/batch_download.py

```diff
@@ -232,6 +232,10 @@ def _get_annotation_metadata(request, search_path, param_list):
         software = [s for s in result_graph.get('software_used', {})]
         software_set = ', '.join([s['software']['title'] for s in software])
         for result_file in result_files:
+            is_file_restricted = restricted_files_present(result_file)
+            is_file_type_excluded = ('files.file_type' in param_list) and (result_file['file_type'] not in param_list['files.file_type'])
```

Comment:
This should be able to be written without '('files.file_type' in param_list)' by using dict.get().  Possibly with a negation in the final condition.

Reply:
Done